src/App.js

```diff
@@ -3,20 +3,23 @@ import './App.scss';
 
 import todos from './api/todos';
 import users from './api/users';
+import { TodoList } from './components/TodoLisst/TodoList';
+
+const preparedTodos = todos.map((todo) => {
+  const user = users.find(person => person.id === todo.userId);
+
+  const copy = { ...todo };
+
+  copy.user = user;
+
+  return copy;
+});
 
 function App() {
   return (
     <div className="App">
-      <h1>Static list of todos</h1>
-      <p>
-        <span>Todos: </span>
-        {todos.length}
-      </p>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
+      <h1 className="App__title">Static list of todos</h1>
+      <TodoList todosArr={preparedTodos} />
```

Comment:
```suggestion
      <TodoList todos={preparedTodos} />
```
better to give the names that actually means what they contains